packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -746,6 +756,17 @@ class SwipeableViews extends Component {
     const touchEvents = disabled ? {} : {
       onTouchStart: this.handleTouchStart,
       onTouchEnd: this.handleTouchEnd,
+      onMouseDown: (e) => {
+        e.preventDefault();
```

Comment:
Why preventing the default? Is it for the text selection? Could you add a comment?